tests/testthat/test-as_latex.R

```diff
@@ -114,7 +144,97 @@ test_that("Table styles correctly applied", {
               locations = cells_stub_grand_summary()) %>%
     tab_style(style = cell_fill(color = '#AAAAAA'),
               locations = cells_grand_summary()) %>%
+    tab_options(latex.use.longtable = TRUE) %>%
     as_latex() %>%
     expect_snapshot()
 
 })
+
+test_that("Table styles correctly applied for tabular*", {
+
+  gt_latex_styled <-
+    gt(exibble[1:5, c(1:3, 5:9)],
+       rowname_col = 'row',
+       groupname_col = 'group') %>%
+    tab_stubhead(label = 'stubhead') %>%
+    tab_stub_indent(rows = everything(),
+                    indent = 'increase') %>%
+    tab_spanner(label = 'heading',
+                id = 'a1',
+                columns = 1:3) %>%
+    tab_spanner(label = 'heading',
+                id = 'b2',
+                columns = 5:6) %>%
+    summary_rows(fns = list(list(label = 'Total', fn = 'sum'),
+                            list(label = 'Avg', fn = 'mean')),
+                 columns = c('num', 'currency'),
+                 groups = 'grp_a',
+                 fmt = ~ fmt_number(., use_seps = TRUE)) %>%
+    grand_summary_rows(fns =list(list(label = 'grand total', fn = 'sum')),
+                       columns = c('num', 'currency'),
```

Comment:
Idem here! You can use styler to restyle the selection in RStudio. https://styler.r-lib.org/